layouts/index.redirects.json

```diff
@@ -12,4 +12,5 @@
 {{- $redirects.SetInMap "paths" . $target -}}
 {{- end -}}
 {{- end -}}
-{{ $redirects.Get "paths" | jsonify }}
+{{- $opts := dict "noHTMLEscape" true }}
+{{- $redirects.Get "paths" | jsonify $opts }}
```

Comment:
This diff is so hard to read, because GitHub tries to render as `json` based on the file-extension, but the diff is;

- adding the `noHTMLEscape` option here
- adding some `-` (strip whitespace) options

correct?

(wondering if an alternative file-extension is possible with Hugo, e.g. `.json.tmpl`, but not sure how that works